### DISCUSSION TITLE
Why doesn't the LXC cron updater also create a snapshot before?

### DISCUSSION THREAD

**Discussion by thibaultmol** [tteck/Proxmox#2228] [Ideas]
I noticed the Update LXC's cron script doesn't take a snapshot before each update. While an update usually should go fine, maybe it's a good idea to give the user the option if they want to have a snapshot for every update?

**Comment by tteck** [comment]
Are you not scheduling backups through the GUI? Ideally, backups should be scheduled at least every 6 hours. While running a snapshot before updates is okay, you need to remember to remove the snapshot from each LXC. In my opinion, it _should_ suffice to have frequent backups in snapshot mode with a moderate retention.

**Reply by thibaultmol** [reply]
Fair point, I guess I was looking at it from the aspect of easily being able to tell when updates happened and roll back to specific moments before an update. But yeah
